### ISSUE TITLE
[Question] what is the appropriate naming convention?

### ISSUE THREAD

**Issue by publicarray** [DNSCrypt/dnscrypt-proxy#533]
What is the appropriate naming convention? These are some that I have found in the wild or from memory.

 - [x] dnscrypt-proxy - This is the repository and binary name
 - [ ] dnscrypt-proxy 2
 - [ ] DNSCrypt-proxy - personally I like this one or the next one
 - [ ] DNSCrypt-Proxy
 - [ ] DNSCrypt-Proxy 2 - This is the (awesome) logo
 - [ ] DNSCrypt proxy (I think this happens only when it is used in GitHub as the title of a wiki page, the dash is interpreted as a space due to how the url is encoded)
 - [ ] other?

Basically I'm asking in order to establish more consistency in the documentation here and elsewhere.

**Comment by jedisct1** [comment]
I tend to use DNSCrypt for the protocol, and dnscrypt-proxy to refer the client, but this is indeed not very consistent with the logo. So... whatever you want :)

**Comment by publicarray** [comment]
Hm I find that even more inconsistent. DNSCrypt where DNS is the acronym and where dnscrypt-proxy doesn't highlight this. I do agree that DNSCrypt is the best capitalisation for the protocol. BTW I always pronounce it "DNS Crypt Proxy" hence why I think the capitalisation could also reflect that. But I also understand that both the binary and the repository are all lower case. I remember v1 as always all lowercase. It's like curl and cURL. but I don't know anyone that actually says C U R L (as individual letters) vs [curl](https://media.merriam-webster.com/soundc11/c/curl0001.wav).

So I think these are three good options: dnscrypt-proxy, DNSCrypt-proxy (`'protocol' + '-proxy'`) or DNSCrypt-Proxy (`'logo' - '2'`)  (adding the `2` is just weird as that is the version number IMHO, I know it's a complete rewrite)

Let me know if you agree or disagree. I don't have a strong opinion just to be more consistent. e.g. [1](https://github.com/jedisct1/dnscrypt-proxy/wiki/dnscrypt-proxy-in-the-media), [2](https://github.com/SynoCommunity/spksrc/wiki/FAQ-DNSCrypt-proxy), [3](https://github.com/pi-hole/pi-hole/wiki/DNSCrypt-2.0), [4](https://github.com/jedisct1/dnscrypt-proxy/wiki)

**Comment by jedisct1** [comment]
Maybe we should stick to `dnscrypt-proxy` for the tool, as it makes it more obvious that the protocol and the tool are different things.

I can update the logo to reflect this.
